06_puppy/willshen8/syncStore_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type SyncStoreTestSuite struct {
+	suite.Suite
+}
+
+func (suite *SyncStoreTestSuite) SetupTest() {
```

Comment:
I may be missing sth, but i cannot find where this fixture was used?
Maybe you can use puppy created by `SetupTest` in `Delete` and `Read` tests

Reply:
I played with that method and didn't find any use-cases for it. So left it in there accidentally.